container/build.py

```diff
@@ -26,45 +42,52 @@
 $ python build.py -d {container_type} -l
 will produce docker locally as {container_type}:latest
 
-usage:
-  build.py -d {rbe-debian8,rbe-debian9,rbe-ubuntu16_04,ubuntu16_04-bazel-docker,ubuntu16_04-bazel}
-      [-p PROJECT] [-c CONTAINER] [-t TAG] [-a] [-b BUCKET] [-h]
-      [-l]
-
 required arguments:
-  -d TYPE, --type TYPE  Type of the container: see TYPE_TARGET_MAP
+  -d, --type            Type of the container: see SUPPORTED_TYPES
+required arguments (for cloud build):
   -p PROJECT, --project PROJECT
                         GCP project ID
   -c CONTAINER, --container CONTAINER
                         Docker container name
   -t TAG, --tag TAG     Docker tag for the image
+  -v BAZEL_VERSION, --bazel_version BAZEL_VERSION
```

Comment:
does this also work with rcX versions of bazel (if so add example, if not make explicit)?

Reply:
It only supports versions supported by our bazel rules, added a reference to the version file.